src/sagas/projects.js

```diff
@@ -47,6 +50,17 @@ export function* changeCurrentProject() {
   yield call(saveCurrentProject, state);
 }
 
+export function* rehydrateProject({payload: {rehydratedProject}}) {
```

Comment:
Now that this function only contains one line, do you think its body should be inlined into `applicationLoaded`?